src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -615,3 +615,122 @@ app.use(function processItems(req, res, next) {
   })
 });
 ```
+
+### Vue.js
+
+The Vue Tracing Integration allows us to track rendering performance during an initial application load.
+
+It is achieved by injecting our handler inside Vue's `beforeCreate` mixin, which gives us access to a Vue component during all of it's lifecycle stages.
+When we encounter component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
+and create a new activity named `Vue Application Render`. Once it is created, it will wait until all of it's child components are rendered, and there are no new rendering events triggered within configured `timeout`, before marking the activity as completed.
+
+This will give you very high-level information about the rendering performance of the Vue instance, however the integration can also provide more fine-grained details about what actually happened during a specific activity.
```

Comment:
```suggestion
`Vue Application Render` will give you very high-level information about the rendering performance of the Vue instance. However, the integration can also provide more fine-grained details about what actually happened during a specific activity.
```

Reply:
"This" = the setup described above, a whole functionality.

Reply:
`Vue Application Render` is just a name of the activity (a conceptual object that keeps track of all the necessary data)

Reply:
Nice. Thanks for the clarification. 😄

Reply:
I checked the updated paragraph on line 627 and it looks good 👍